### DISCUSSION TITLE
offline Libraries

### DISCUSSION THREAD

**Discussion by dwk000** [community/community#22273] [New to GitHub]
<p>Hi,</p>
<p>  Is there anyway of doing a Library dump of GitHub</p>
<p>  ie a dump of all webcam Libraries</p>
<p>  so that a Library can be install offline later</p>
<p>DK</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>There’s no automated way of doing what you want, no. But you could write a script or an app that would search GitHub for all repositories matching your specifications, then clone each repository locally or download the latest release.</p>
<p>I hope that helps!</p>
